checkbox-core-snap/series16/snap/snapcraft.yaml

```diff
@@ -113,6 +116,8 @@ parts:
       - libcap-dev
       - libsctp-dev
       - libjudy-dev
+      - libjudydebian1
```

Comment:
I don't think you need to add libjudydebian1 and libsctp1 to build-packages. If snapcraft issued a warning, the proprosal was to add the missing lib in the stage-packages instead

Reply:
Fixed it, and tested it by build snap again.